File: module7/Collaboration/Python/CallRoom.py
import requests

url = "https://api.tropo.com/1.0/sessions"

querystring = {"action":"create","token":"<your Tropo App token>","numbertodial":"<your Spark room URI>"}

headers = {
    'cache-control': "no-cache",
    'postman-token': "73dad72f-0962-de88-8701-73557b38315b"
    }

response = requests.request("GET", url, headers=headers, params=querystring)

print(response.text)

File: module7/Collaboration/NodeJS/CallRoom.js
var request = require("request");

var options = { method: 'GET',
  url: 'https://api.tropo.com/1.0/sessions',
  qs: 
   { action: 'create',
     token: '<your Tropo app token>',
     numbertodial: '<your Spark room URI>' },
  headers: 
   { 'postman-token': '21bcef21-2858-ee62-97a9-5f0634c63524',
     'cache-control': 'no-cache' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
